test/e2e/create_cluster_test.go

```diff
@@ -1299,6 +1331,9 @@ func TestCreateClusterCustomConfig(t *testing.T) {
 		e2eutil.EnsureSecretEncryptedUsingKMSV2(t, ctx, hostedCluster, guestClient)
 		// test oauth with identity provider
 		e2eutil.EnsureOAuthWithIdentityProvider(t, ctx, mgtClient, hostedCluster)
+
+		// ensure image registry component is disabled
+		testCreateClusterWithDisabledCapabilities(ctx, t, g, mgtClient, hostedCluster)
```

Comment:
Could you follow the same approach we do putting the `testCreateClusterWithDisabledCapabilities` function into the e2eutil pkg and name it as we do by convention?